spec/models/account_migration_spec.rb

```diff
@@ -83,7 +87,8 @@
       end
 
       context "with local old user" do
-        include_context "with local old user"
+        let(:old_user) { FactoryGirl.create(:user) }
```

Comment:
Here only `..._person` is used, so it can be changed to `FactoryGirl.create(:user).person` and `..._user` can be removed.